validator/accounts/accounts_backup.go

```diff
@@ -105,15 +104,6 @@ func BackupAccountsCli(cliCtx *cli.Context) error {
 		if err != nil {
 			return errors.Wrap(err, "could not backup accounts for local keymanager")
 		}
-	case keymanager.Derived:
```

Comment:
one thing to consider is wallet migration, those who have the derived type should be parsed and swapped to local somehow with the flag if we want to go with it this way 🤔

Reply:
please look at [validator/keymanager/types.go](https://github.com/prysmaticlabs/prysm/pull/10211/files#diff-ac8b5b545c26841cf2c80a7d61aefbbc6c897d1648a3196ef1f631636d29fa91) and how parsing works. one worry that I have is the possibility of having a direct/derived directory locally for an existing client. we need a way to safely merge or handle this for existing users. otherwise it will only use the direct file( representing imported) instead of realizing there is a derived filed at the wallet dir.